port/linux/omrosdump_helpers.c

```diff
@@ -183,7 +190,7 @@ renameDump(struct OMRPortLibrary *portLibrary, char *filename, pid_t pid, int si
 		portLibrary->str_printf(portLibrary,
 								filename,
 								EsMaxPath,
-								"Error opening \"%s\". Look for core file generated by child process with pid = %i",
+								"Error opening \"%s\".  Look for core file generated by child process with pid = %i",
 								coreUsesPIDFileName,
 								pid);
```

Comment:
Continued lines should be indented two tabs more than the beginning of the statement; same comment for the multi-line calls below beginning on lines 236 & 245.

Reply:
Ahh yes, I can fix that, and also I'll move `portLibrary` to a newline. However, just to clarify, shouldn't the tabs align with the opening bracket?

> Where reasonable, the parameter list should be aligned with the opening bracket.

https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line

Reply:
I'm not sure it would be "reasonable" in this situation. You'll also note that all of the promoted examples indent two tabs.

Reply:
Makes sense. I'm back in the office Monday and will update then.